subprojects/gradle/instrumentation-test-impact-analysis/src/main/kotlin/com/avito/instrumentation/impact/InstrumentationTestImpactAnalysisPlugin.kt

```diff
@@ -26,9 +24,6 @@ class InstrumentationTestImpactAnalysisPlugin : Plugin<Project> {
         val extension = project.extensions
             .create<InstrumentationTestImpactAnalysisExtension>("instrumentationTestImpactAnalysis", project)
 
-        //todo should work in any environment
-        if (project.buildEnvironment !is BuildEnvironment.CI) return
```

Comment:
We were failing cause of that in IDE sync phase and it configuration phase without ci=true, cause instrumentation implicitly depends on that plugin